packages/contracts/test/contracts/OVM/execution/OVM_ExecutionManager.gas-spec.ts

```diff
@@ -117,5 +148,123 @@ describe('OVM_ExecutionManager gas consumption', () => {
+        returnData:
+          '0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000201111111111111111111111111111111111111111111111111111111111111111',
+      },
+      {
+        inputData:
+          '0x11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
+        returnData:
+          '0x0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000004011111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
+      },
+      {
+        inputData:
+          '0x111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
+        returnData:
+          '0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000060111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
+      },
+      {
+        inputData:
+          '0x1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
+        returnData:
+          '0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000801111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
+      },
+    ]
+
+    dataVariants.forEach(async (dataVariant) => {
+      it('Gas cost of ovmCALL', async () => {
+        const ovmCallData = Helper_ExecutionManager.interface.encodeFunctionData(
+          'returnData',
+          [dataVariant.inputData]
+        )
+
+        const encodedStep = Helper_TestRunner.interface.encodeFunctionData(
+          'runSingleTestStep',
+          [
+            {
+              functionName: 'ovmCALL',
+              functionData: ovmCallData,
+              expectedReturnStatus: true,
+              expectedReturnData: dataVariant.returnData,
+              onlyValidateFlag: false,
+            },
+          ]
+        )
+
+        const tx = await Helper_ExecutionManager.ovmCALLHelper(
+          OVM_TX_GAS_LIMIT,
+          Helper_TestRunner.address,
+          encodedStep,
+          OVM_StateManager.address,
+          { gasLimit: RUN_OVM_TEST_GAS }
+        )
```

Comment:
I think it would be very helpful to future collaborators to included a comment explaining these steps does.
Looks to me like: 
1. prepend the selector for `returnData(bytes)` to the inputData
2. encode that as call data to pass to `Helper_TestRunner.runSingleTestStep()`
3. Call the `Helper_ExecutionManager` with `Helper_TestRunner` as target, which call back to the `Helper_ExecutionManager`'s `returnData` function.